jetstream/sponsored-suggestion-placement-phase-2.toml

```diff
@@ -0,0 +1,694 @@
+[experiment]
+
+enrollment_period = 14
```

Comment:
Should be at least 20, to cover the larger range `BETWEEN '2024-01-17' AND '2024-02-06'` from cohort 2.